st2common/tests/integration/test_rabbitmq_ssl_listener.py

```diff
@@ -58,14 +55,19 @@ def test_non_ssl_connection_on_ssl_listener_port_failure(self):
 
         expected_msg_1 = '[Errno 104] Connection reset by peer'
         expected_msg_2 = 'Socket closed'
+        expected_msg_3 = 'Server unexpectedly closed connection'
+        epxected_msg_4 = '[Errno 104] ECONNRESET'
 
         try:
             connection.connect()
         except Exception as e:
             self.assertFalse(connection.connected)
             self.assertTrue(isinstance(e, (IOError, socket.error)))
-            self.assertTrue(expected_msg_1 in six.text_type(e) or expected_msg_2 in
-                            six.text_type(e))
+            self.assertTrue(expected_msg_1 in six.text_type(e) or
```

Comment:
Not a huge fan of this. It's fine for now, but if gets more complicated I think a good combination of a list comprehension and set intersection might be a better way to implement this.

Reply:
seems like every platform has a slightly different error message. I had to add one for xenial, and then bionic had yet another. :/

Reply:
Yeah, it's fine for now, it's just something we should keep in mind in the future.